### PR TITLE
NG1256 - Fix for Datepicker not clearing in angular version.

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR will fix a bug on datepicker not clearing in angular version.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise-ng/issues/1256

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go http://localhost:4000/components/datepicker/test-input-datefield-clear.html
- Click `Clear Field`

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -4,14 +4,19 @@
...
+- `[Datepicker]` Fixed a bug on setValue() when pass an empty string for clearing field. ([#6168](https://github.com/infor-design/enterprise/issues/6168))
+- `[Datepicker]` Fixed a bug on datepicker not clearing in angular version. ([NG#1256](https://github.com/infor-design/enterprise-ng/issues/1256))
+- `[Dropdown]` Fixed on keydown events not working when dropdown is nested in label. ([NG#1262](https://github.com/infor-design/enterprise-ng/issues/1262))
 - `[Listview]` Added flex toolbar for multiselect listview. ([NG#1249](https://github.com/infor-design/enterprise-ng/issues/1249))
 - `[Listview]` Adjusted spaces between the search icon and filter wrapper. ([#6007](https://github.com/infor-design/enterprise/issues/6007))
+- `[Listview]` Changed the font size of heading, subheading, and micro in Listview Component. ([#4996](https://github.com/infor-design/enterprise/issues/4996))
+- `[Modal]` Fixed on too wide minimum width when close button is enabled. ([NG#1240](https://github.com/infor-design/enterprise-ng/issues/1240))
+- `[Slider]` Fixed background color of slider in a modal in new dark theme. ([6211](https://github.com/infor-design/enterprise-ng/issues/6211))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md:18]: Seems a lot more added here then expected? Not sure why?

**Review by tmcconechy — DISMISSED** [review]

**Review by yohannahbautista — APPROVED** [review]

File: docs/CHANGELOG.md (hunk 1 above)
Comment (tjamesallen15) [reply docs/CHANGELOG.md:18]: > Seems a lot more added here then expected? Not sure why? The only question i have - maybe resync the change log

Yeah. I'll update the change.

**Review by yohannahbautista — DISMISSED** [review]

**Comment by tjamesallen15** [comment]
Done fixing merge conflicts. Wanted to merge this one so that we can permanently update the changelog from present tense to past tense.

**Comment by tjamesallen15** [comment]
This is also needed for this ticket in NG. https://github.com/infor-design/enterprise-ng/issues/1256

**Review by tmcconechy — APPROVED** [review]

**Review by ericangeles — APPROVED** [review]

**Pull request merged**